package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autorest/python",
-  "version": "5.1.0-preview.7",
+  "version": "5.2.0-preview.8",
```

Comment:
decided to bump a minor version since this is a breaking change and a lot of files get changed too